src/codex_plugin_scanner/guard/daemon/static/assets/index.css

```
@@ -4454,8 +4454,22 @@
     text-decoration-line: underline;
   }
 
-  .underline-offset-2 {
-    text-underline-offset: 2px;
+  .decoration-brand-blue\/45 {
+    -webkit-text-decoration-color: var(--brand-blue);
```

Stylelint (17.14.0)

[error] Duplicate property "-webkit-text-decoration-color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)